SPECS/glslang/glslang.spec

```diff
@@ -0,0 +1,78 @@
+%global sdkver 1.3.275.0
+
+Name:           glslang
+Version:        14.0.0
+Release:        1%{?dist}
+Summary:        OpenGL and OpenGL ES shader front end and validator
+
+License:        BSD and GPLv3+ and Apache-2.0
+URL:            https://github.com/KhronosGroup/%{name}
+Source0:        %url/archive/vulkan-sdk-%{sdkver}.tar.gz#/%{name}-sdk-%{sdkver}.tar.gz
+# Patch to build against system spirv-tools (rebased locally)
+#Patch3:         https://patch-diff.githubusercontent.com/raw/KhronosGroup/glslang/pull/1722.patch#/0001-pkg-config-compatibility.patch
+Patch3:         0001-pkg-config-compatibility.patch
+
+BuildRequires:  cmake3
+BuildRequires:  gcc-c++
+BuildRequires:  ninja-build
+BuildRequires:  spirv-tools-devel
+
+%description
+%{name} is the official reference compiler front end for the OpenGL
+ES and OpenGL shading languages. It implements a strict
+interpretation of the specifications for these languages.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+%{name} is the official reference compiler front end for the OpenGL
+ES and OpenGL shading languages. It implements a strict
+interpretation of the specifications for these languages.
+
+%prep
+%autosetup -p1 -n %{name}-vulkan-sdk-%{sdkver}
+# Fix rpmlint warning on debuginfo
+find . -name '*.h' -or -name '*.cpp' -or -name '*.hpp'| xargs chmod a-x
+
+%build
+%cmake3 -DBUILD_SHARED_LIBS=OFF
```

Comment:
Can we keep the shared libs on?  Do we need to produce the static libs?  (We try to avoid static libraries as much as possible because it causes issues with CVE maintenance...tracking down who used the static libraries and then rebuilding all the affected packages...This is especially difficult if external customers use the static libs)

Reply:
Done. Added all .so's to the main package.

Reply:
Please see the last two commits that address your feedback